src/dashboard/index.ts

```diff
@@ -45,28 +42,24 @@ export default function () {
     const docker = new Docker(mainWindow!)
     const firefox = new Firefox(mainWindow!)
     ipcMain.on('firefox:check', async (_, message) => {
-      
       const firefoxInstalled = await firefox.isInstalled()
       if (!firefoxInstalled) {
-        await firefox.download();
-      }
-      else{
-        await firefox.launch();
+        await firefox.download()
+      } else {
+        await firefox.launch()
       }
     })
 
     ipcMain.on('firefox:lunch', async (_, message) => {
```

Comment:
Should be `firefox:launch` ipc event name?

Reply:
Another typo